### PR TITLE
Move to pnpm

### PR DESCRIPTION
**Pull request by qjack001**
Let's see if this even builds...

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -1,7 +1,10 @@
 [build]
-	command = "npm run build"
+	command = "npx pnpm@6 install --shamefully-hoist --store-dir=.pnpm --frozen-lockfile && npx pnpm@6 build"
 	publish = "dist"
 
+[build.environment]
+	NODE_VERSION = "16"
```
Comment (qjack001) [inline netlify.toml:6]: `.node-version` can be removed in favour of an environment variable

**Comment by qjack001** [comment]
why does netlify never do a deployment preview of this one?

**Comment by HoloPollock** [comment]
Oh it's funny it does it it just doesn't comment it

**Comment by HoloPollock** [comment]
https://deploy-preview-17--vivicresearch.netlify.app

**Review by HoloPollock — APPROVED** [review]

**Pull request merged**